src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/policy/AdditionalConditionPss.kt

```diff
@@ -6,6 +6,7 @@ data class AdditionalConditionPss(
   override val text: String,
   override val tpl: String?,
   override val requiresInput: Boolean,
+  override val categoryShort: String?,
```

Comment:
To help @SebNegahbanMoJ work with conditions using polymorphism.

( sounds like there's no good reason to not include this from a modelling point of view)